.husky/pre-push

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+branch_name=$(git rev-parse --abbrev-ref HEAD)
+pattern="^FS-[0-9]+/[a-z0-9._-]+$"
+
+if ! [[ $branch_name =~ $pattern ]]; then
+  echo "Branch name '$branch_name' does not follow the naming convention 'FS-<number>/<branch-name>', please rename and try again."
```

Comment:
Could be worth printing the pattern as well. I like that you have prevented uppercase letters, it would be helpful to print the pattern in case someone doesn't realise